Nodejs/Product/TargetsVsix/Microsoft.NodejsToolsV2.targets

```diff
@@ -0,0 +1,313 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Specifies the directories which are ignored.  Any paths which contain these directory names
+         will be ignored.  The directory name must completely match and no wild cards are allowed
+     -->
+    <AnalysisIgnoredDirectories Condition="'$(AnalysisIgnoredDirectories)' == ''">bower_components;node_modules\core-js</AnalysisIgnoredDirectories>
+    <DockerExtraPublishProfileProperties>ExcludeFoldersFromDeployment=node_modules</DockerExtraPublishProfileProperties>
```

Comment:
node_modules but not bower_components?

Reply:
This is copied from the NodejsTools.targets file. I'm not very familiar with bower it might be necessary to copy this in deployment.